addons/sourcemod/scripting/vsh/abilities/ability_rage_ghost.sp

```diff
@@ -83,6 +109,8 @@ methodmap CRageGhost < SaxtonHaleBase
 		ability.flDuration = 8.0;
 		ability.flHealSteal = 20.0;	//Steals hp per second
 		ability.flHealGain = 40.0;	//Gains hp per second
+		ability.flBuildingDrain = 1.0;	//Building health drain multiplier, 0.0 or lower disables it
```

Comment:
Add note it scales from `flHealSteal`, i got confused from that when reading through.

Reply:
I did that after making this pull request locally but forgot to update it in here, oops